docs/guides/web/nginx-mainline.md

```diff
@@ -65,19 +65,19 @@ From there, you could just start dropping HTML files into the `/usr/share/nginx/
 
 !!! Note
 
-    Old versions of this guide used to detail how to install the nginx-mainline package from ELRepo. This is no longer an option, however. In most cases, the version of Nginx in Rocky's repos is more than sufficient, providing a stable base with backported security patches. Those who still want to use the nginx-mainline branch can follow [this guide from LinuxCapable](https://www.linuxcapable.com/how-to-install-nginx-mainline-on-rocky-linux/). Please note that using nginx-mainline is usually perfectly viable, but not supported.
+    У старих версіях цього посібника детально описано, як встановити пакет nginx-mainline з ERLepo. This is no longer an option, however. In most cases, the version of Nginx in Rocky's repos is more than sufficient, providing a stable base with backported security patches. Those who still want to use the nginx-mainline branch can follow [this guide from LinuxCapable](https://www.linuxcapable.com/how-to-install-nginx-mainline-on-rocky-linux/). Please note that using nginx-mainline is usually perfectly viable but not supported.
```

Comment:
@gannazhyrnova OOps! Looks like on line 68 you've managed to add translated text to the line instead of correcting what you wanted to corect. Can you review and change this?

Reply:
@gannazhyrnova I fixed this (I hope I used words that you would use) so that it could be merged. Thank you!